.github/workflows/code-style.yml

```diff
@@ -37,3 +37,18 @@ jobs:
         uses: py-actions/flake8@v1
         with:
           path: "pycrostates"
+      - name: Run black
+        uses: psf/black@stable
+        with:
+          options: "--check --verbose"
+      - name: Run isort
+        uses: jamescurtin/isort-action@master
+        with:
+            requirementsFiles: "requirements.txt requirements_testing.txt"
+      - name: Install pydocstyle
+        run: | 
+          python -m pip install -r requirements_testing.txt
```

Comment:
Line 30:33 Install flake8
If you do have flake8 in `requirements_testing.txt`. which you should, then instead of doing a second installation step just rename to `Install dependencies` and uses this earlier instead of `Install flake8`. You can also add the upgrade of pip, setuptools and wheel and the `--progress-bar off` to reduce verbovsity.